src/NodaTime.Demo/LocalDateDemo.cs

```diff
@@ -104,5 +106,82 @@ public void Min()
             LocalDate min = Snippet.For(LocalDate.Min(earlyJune, lateJune));
             Assert.AreEqual(new LocalDate(2010, 6, 5), min);
         }
+
+        [Test]
+        public void FromDateTime()
+        {
+            DateTime earlyJune = new DateTime(2010, 6, 5);
+            LocalDate date = Snippet.For(LocalDate.FromDateTime(earlyJune));
+            Assert.AreEqual(new LocalDate(2010, 6, 5), date);
+        }
+
+        [Test]
+        public void FromDateTimeWithCalendarSystem()
+        {
+            DateTime earlyJune = new DateTime(2010, 6, 5);
+            CalendarSystem calendar = CalendarSystem.ForId("Julian");
+            LocalDate date = Snippet.For(LocalDate.FromDateTime(earlyJune, calendar));
+            //Until 2099, Julian calendar is 13 days behind Gregorian
+            Assert.AreEqual(2010, date.Year);
+            Assert.AreEqual(5, date.Month);
+            Assert.AreEqual(23, date.Day);
+        }
+
+        [Test]
+        public void CommonEra()
+        {
+            LocalDate date = Snippet.For(new LocalDate(Era.Common, 2010, 6, 16));
```

Comment:
Currently we only support one snippet per member (the constructor accepting an Era in this case).
I suspect the BeforeCommonEra would be a better example in this case, so it's worth removing this one.

Reply:
I thought that might be the case but you did state you were looking at some enhancements for the snippet handling!